test/ng/compileSpec.js

```diff
@@ -4193,7 +4194,7 @@ describe('$compile', function() {
       dealoc(element);
       compile('<div><span my-component>');
       expect(componentScope).not.toBe(firstComponentScope);
-      expect(componentScope.$$isolateBindings).toBe(first$$isolateBindings);
+      expect(componentScope.$$isolateBindings).toEqual(first$$isolateBindings);
```

Comment:
this is the comparison of the object that contains all the bindings. In this case, the object will be different but the content of both objects will have the same object instances as content.